ngraph/core/include/ngraph/op/round.hpp

```diff
@@ -54,5 +56,64 @@ namespace ngraph
         NGRAPH_SUPPRESS_DEPRECATED_START
         using v0::Round;
         NGRAPH_SUPPRESS_DEPRECATED_END
+
+        namespace v5
+        {
+            /// \brief Elementwise round operation. The output is round to the nearest integer
+            /// for each value. In case of halfs, the rule is defined in attribute 'mode':
```

Comment:
Replace single quote with double quote. It should resolve the lint error that you have now.

Reply:
The problem was in another place, now it's fixed.